src/main/java/io/clue2solve/aws/bedrock/springboot/starter/autoconfigure/aimodels/OnTitan.java

```diff
@@ -3,37 +3,61 @@
 import org.springframework.boot.autoconfigure.condition.AnyNestedCondition;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
 
+/**
+ * Condition class for Titan
+ */
 class OnTitan extends AnyNestedCondition {
```

Comment:
Pay attention to each `On*` condition name.  Adjust the "Condition for" to match the exact Titan model variant.  Thanks!